### PR TITLE
update AccessKit for new dashboard

### PR DESCRIPTION
**Pull request by AprilSylph**
Fixes options:
 - Post font
 - Make links blue
 - Don't display NPF colors
 - Increase post margins

Also rewrites the inverted and grayscale colours options as to include them in the same new block.

I'm considering this a full fix of the extension for the React site, since
 - there are no longer hidden image captions
 - dashboard contrast is now good by default
 - the font size is now bigger
 - the XKit options that AccessKit disables are Tweaks that haven't been fixed yet

### CODE REVIEW

File: Extensions/accesskit.js [hunk 1]
```diff
@@ -42,28 +42,28 @@ XKit.extensions.accesskit = new Object({
 		},
 		no_npf_colors: {
 			text: "Don't display NPF (user-set) colours in posts",
-			default: true,
-			value: true
+			default: false,
```
Comment (hobinjk) [inline Extensions/accesskit.js:45]: Good defaults to have 👍

File: Extensions/accesskit.js [hunk 2]
```diff
@@ -108,9 +108,69 @@ XKit.extensions.accesskit = new Object({
...
+				);
+			}
+
+			if (xkit_contrast_icons.value) {
+				XKit.tools.add_css(this.xkit_contrast_icons_css, 'accesskit');
+			}
+
+			const {invert, grayscale} = this.preferences;
+			if (invert.value || grayscale.value) {
+				const do_invert = invert.value ? "invert(100%)" : "";
```
Comment (hobinjk) [inline Extensions/accesskit.js:157]: Double quotes when there are mostly single quotes above 🔪

File: Extensions/accesskit.js [hunk 3]
```diff
@@ -108,9 +108,69 @@ XKit.extensions.accesskit = new Object({
 		}
```
Comment (hobinjk) [inline Extensions/accesskit.js:108]: Would be nice to have a thing on `contrast` above saying it's deprecated or something

File: Extensions/accesskit.js (hunk 2 above)
Comment (AprilSylph) [reply Extensions/accesskit.js:157]: :pensive: my habits strike again...

File: Extensions/accesskit.js (hunk 3 above)
Comment (AprilSylph) [reply Extensions/accesskit.js:108]: i was thinking of hiding the deprecated options on the react site, so ill give that a proper try

Comment (hobinjk) [reply Extensions/accesskit.js:108]: Neato burrito

**Pull request merged**